test/metric_learn_test.py

```diff
@@ -76,6 +76,111 @@ def test_singular_returns_pseudo_inverse(self):
                     pseudo_inverse)
 
 
+class TestSCML():
```

Comment:
maybe better to inherit from `object` explicitly, see
https://stackoverflow.com/questions/15374857/should-all-python-classes-extend-object